docs/branching-merging.md

```diff
@@ -4,7 +4,7 @@ Branches should be named descriptively, preferably in some way that indicates wh
 
 All pull requests to merge into master must be reviewed by at least one member of the cf.gov platform team. The cf.gov platform team will ensure that these reviews happen in a timely manner. To ensure timely code reviews, please tag all PRs to master with @cfpb/cfgov-backends and @cfpb/cfgov-frontends as appropriate.
 
-When reviewing pull requests, it is important to note explicit blockers from things that can be addressed later or would be nice to have. This is best as a simple top-level post after review to summarize the review.
+When reviewing pull requests, it is important to distinguish between explicit blockers and things that can be addressed later or would be nice to have. This is best as a simple top-level post after review to summarize the review.
```

Comment:
You could add here that `TODO:` comment can be added for anything that isn't strictly a show stopper, but should be addressed in the future.